packages/rs-drive-abci/src/execution/types/execution_operation/signature_verification_operation.rs

```diff
@@ -26,3 +33,51 @@ impl OperationLike for SignatureVerificationOperation {
         Ok(0)
     }
 }
+
+pub fn signature_verification_operations_from_state_transition(
+    state_transition: &StateTransition,
+    identity: Option<&PartialIdentity>,
+) -> Option<Vec<SignatureVerificationOperation>> {
+    match state_transition {
+        StateTransition::IdentityCreate(state_transition) => Some(
+            state_transition
+                .public_keys()
+                .iter()
+                .map(|key| SignatureVerificationOperation::new(key.key_type()))
+                .collect(),
+        ),
+        StateTransition::IdentityUpdate(state_transition) => {
+            let mut operations = vec![];
+
+            let identity = identity.unwrap();
+            let key = identity
+                .loaded_public_keys
+                .get(&state_transition.signature_public_key_id())
+                .unwrap();
```

Comment:
don't unwrap in production code.

Reply:
Do we really need to introduce `Result` to this function? 
We are expecting signer identity to posses `signature_public_key_id`. If it doesn't, then something radically wrong is happening.